charts/prometheus-kafka-exporter/templates/deployment.yaml

```
@@ -80,10 +80,9 @@ spec:
             - {{ . }}
           {{- end }}
           {{- end }}
+          {{- with .Values.env }}
           env:
-          {{- range $env := .Values.env }}
-            - name: {{ $env.name }}
-              value: {{ $env.value }}
+            {{- toYaml . | nindent 12}}
```

Thank you, @marcfrederick, for your PR. 

```suggestion
            {{- toYaml . | nindent 12 }}
```

I would suggest to add a comment/example in the values file that e.g. a secret/cm can also be used. For example

```yaml
  ## env sets environment variables to pass to the container.
  ## Can be set as name/value pairs, read from secrets or configmaps.
  env: []
    # - name: <ENV_VAR_NAME>
    #   value: <value>
    # - name: PASSWORD
    #   valueFrom:
    #     secretKeyRef:
    #       name: mysecret
    #       key: password
    #       optional: false
```

---

I've pushed an updated commit incorporating these changes